symplyphysics/core/vectors/arithmetics.py

```diff
@@ -196,7 +203,8 @@ def reject_cartesian_vector(
     )
 
 
-def diff_cartesian_vector_components(
+# Only works in non-rotating coordinate systems
+def diff_cartesian_vector(
```

Comment:
Same for integration. Also, since you removed `components` from diff name, maybe you should do the same for integration?

Reply:
Makes sense.